torch/fx/proxy.py

```diff
@@ -142,6 +142,20 @@ def create_node(self, kind : str, target : Target,
             for field in copy_meta_fields:
                 if field in current_meta:
                     node.meta[field] = current_meta[field]
+
+            # Here we decrement to account for the sequence_nr having
+            # just been incremented while tracing this lowered aten op.
+            new_seq_nr = torch.autograd.get_sequence_nr() - 1
+            # In the FWD pass the new_seq_nr number maintained by
+            # torch.autograd.get_sequence_nr() keeps increasing.
+            # During the bwd pass it should stay the same value as
+            # the max_seq_nr. The seq_nr for bwd ops is maintained
+            # in the torch::autograd::Node class see NOTE [ Sequence Number ].
+            if current_meta.get("bwd_op", False):
+                new_seq_nr = current_meta["seq_nr"]
+                current_meta["bwd_op"] = False
+            node.meta["seq_nr"] = new_seq_nr
+
```

Comment:
I finally got setting node.meta["seq_nr"] isolated to 1 place.
I added a field to current_meta to call out bwd_ops.  Then in create_node() the seq_nr for bwd ops is taken from current_meta and for fwd ops it is taken from get_sequence_nr().